packages/uniforms-antd/__tests__/index.ts

```diff
@@ -28,3 +29,14 @@ it('exports everything', () => {
     wrapField: expect.any(Function),
   });
 });
+
+describe('@RTL', () => {
+  suites.testAutoField(antd.AutoField);
+  // FIXME: AntD `DatePicker` is far from the HTML one.
+  // suites.testDateField(antd.DateField);
+  suites.testListDelField(antd.ListDelField);
+  suites.testListField(antd.ListField, {
+    getListAddField: screen => screen.getByRole('img', { name: 'plus-square' }),
+  });
+  suites.testTextField(antd.TextField);
```

Comment:
I'm not sure if I like moving suite tests to the index files while we still have some tests in a separate file.

Reply:
Got it. I wanted to do it this way just to force us into centralizing this "config" and improving on it later... Let's discuss it and if needed, I'll move it back.